### ISSUE TITLE
How to encode a whitened-out ac reading?

### ISSUE THREAD

**Issue by ppasedach** [chchch/upama#13]
`<subst><del><unclear reason="white correction fluid applied"> </unclear></del><add>something</add></subst>` kind of works, but it's not quite correct, as the ac reading is not really a space, that's only the intermediate step before something could be written again.

**Comment by wujastyk** [comment]
I'm also not certain, but <gap> looks promising:

> By its nature, the gap element has no content. It marks a point in the text where nothing at all can be read, whether because of authorial or scribal erasure, physical damage, or any other form of illegibility. Its attributes allow the encoder to specify the amount of text which is illegible in this way at this point, using any convenient units, where this can be determined.

It takes @reason and @agent.

See further [TEI 11.3.1.7](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/PH.html).

**Comment by chchch** [comment]
Yes I've been using `<gap>` as well! Like this:

`<subst><del rend="white correction fluid applied"><gap reason="illegible"></gap></del><add>something</add></subst>`

But as I mentioned before, I've been vaguely misusing `@rend` and `@reason`; I write prose descriptions where TEI specifies keywords. But I don't think it's a big issue at the moment.

**Comment by wujastyk** [comment]
I agree (prose in place of keyword, to be sorted out at some future time if and when a pattern emerges).
